src/lib/utils/meta_tables/meta_system_information_table.cpp

```diff
@@ -0,0 +1,89 @@
+#ifdef __linux__
+#include <numa.h>
+#include <sys/sysinfo.h>
+#endif
+
+#include <fstream>
+
+#include "meta_system_information_table.hpp"
+
+namespace opossum {
+
+MetaSystemInformationTable::MetaSystemInformationTable()
+    : AbstractMetaSystemTable(TableColumnDefinitions{{"cpu_count", DataType::Int, false},
+                                                     {"system_memory_total_bytes", DataType::Long, false},
+                                                     {"numa_cpu_count", DataType::Int, false},

```

Comment:
Had to look into the code to find out what this means. This doesn't seem to be related to NUMA. Maybe `cpu_affinity_count`?